server/schedulers/utils.go

```diff
@@ -119,24 +121,24 @@ func (p *balancePlan) shouldBalance(scheduleName string) bool {
 
 func (p *balancePlan) getTolerantResource() int64 {
 	if p.kind.Resource == core.LeaderKind && p.kind.Policy == core.ByCount {
-		tolerantSizeRatio := p.cluster.GetOpts().GetTolerantSizeRatio()
-		if tolerantSizeRatio == 0 {
-			tolerantSizeRatio = leaderTolerantSizeRatio
-		}
-		leaderCount := int64(1.0 * tolerantSizeRatio)
-		return leaderCount
+		return int64(p.tolerantSizeRatio)
 	}
-
 	regionSize := p.region.GetApproximateSize()
 	if regionSize < p.cluster.GetAverageRegionSize() {
 		regionSize = p.cluster.GetAverageRegionSize()
 	}
-	regionSize = int64(float64(regionSize) * adjustTolerantRatio(p.cluster))
-	return regionSize
+	return int64(float64(regionSize) * p.tolerantSizeRatio)
```

Comment:
can we move them to `adjustTolerantRatio`？

Reply:
The `regionSize` is related to the region which is selected to balance so it cannot be calculated ahead.